src/Entity/User.php

```diff
@@ -31,7 +38,7 @@ class User extends AbstractBaseEntity implements UserInterface, PasswordAuthenti
     /**
      * @var string The hashed password
      *
-     * @ORM\Column(type="string")
+     * @ORM\Column(type="string", nullable=true)
      */
     private string $password = '';
```

Comment:
Either let it be nullable and change the default value to `null` or leave as is, non-nullable and empty string

Reply:
Changed back